docker/sudo-opensuse/Dockerfile

```diff
@@ -1,9 +1,14 @@
-FROM opensuse:13.2
-MAINTAINER samvanoort@gmail.com
+FROM opensuse/leap:15.1
+MAINTAINER mark.earl.waite@gmail.com
```

Comment:
```suggestion
```

Reply:
Resolved in 77b3dfb49fba0446dc77c927fa5ac46b5aea9d54